src/main/java-templates/utils/ProcessorSolutionInfo.java

```diff
@@ -0,0 +1,19 @@
+package com.commercetools.paymenttoorderprocessor.utils;
```

Comment:
Package name doesn't match with the path?

Reply:
6bfe054